tools/run-wpt.mjs

```diff
@@ -32,7 +32,16 @@ function log(message) {
 
 function usage() {
   log(
-    `Usage: [CHROMEDRIVER=<true | false>] [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] ${process.argv[1]} [webdriver/tests/bidi/[...]]`
+    `Usage:
+      [CHROMEDRIVER=<true | default: false>]
+      [HEADLESS=<true | default: false>]
+      [MANIFEST=<default: 'MANIFEST.json'>]
+      [RUN_TESTS=<default: true | false>]
+      [UPDATE_EXPECTATIONS=<true | default: false>]
+      [VERBOSE==<true | default: false>]
+      [WPT_REPORT=<default: 'wptreport.json'>]
+      [WPT_METADATA=<default: 'wpt-metadata/$\{ 'chromedriver' | 'mapper' }/$\{ 'headless' | 'headful' }>]
+      ${process.argv[1]} [webdriver/tests/bidi/[...]]`
```

Comment:
@jrandolf I'm not sure about the help format, but it's not critical. If you think it should be written in another way, feel free to add edit suggestion.

Reply:
This version looks like:
```
      [CHROMEDRIVER=<true | default: false>]
      [HEADLESS=<true | default: false>]
      [MANIFEST=<default: 'MANIFEST.json'>]
      [RUN_TESTS=<default: true | false>]
      [UPDATE_EXPECTATIONS=<true | default: false>]
      [VERBOSE==<true | default: false>]
      [WPT_REPORT=<default: 'wptreport.json'>]
      [WPT_METADATA=<default: 'wpt-metadata/${ 'chromedriver' | 'mapper' }/${ 'headless' | 'headful' }>]
      chromium-bidi/tools/run-wpt.mjs [webdriver/tests/bidi/[...]]
```